### ISSUE TITLE
New Customizer - tabbing not as expected yet

### ISSUE THREAD

**Issue by Guido07111975** [ClassicPress/ClassicPress#2557]
This is about the updated Customizer in Nightly.

### Description
I saw 2 tabbing related things that don't work as expected yet.

1. When tabbing the Customizer sidebar it does not jump to the preview (the theme pencils). Stays in the sidebar.
2. When adding a widget (or menu item) the sub panel closes (as it should be), but when tabbing further it jumps to the preview (see screenshots). It should stay in the sidebar, jump back to the "Add a widget" (or "Add items") button for example.

### Adding widget
<img width="620" height="255" alt="Image" src="https://github.com/user-attachments/assets/2f9d166b-1815-4a7e-bd31-06cb688e704e" />

### Jumps to preview
<img width="495" height="484" alt="Image" src="https://github.com/user-attachments/assets/0dccfc2b-63d8-4c56-b1ae-e82241865122" />

### ClassicPress version
Current version: 2.7.0+nightly.20260624

### PHP version
8.2

**Comment by KTS915** [comment]
1 is actually exactly as it should be. This is extremely important for accessibility and is a deliberate design change.
2 is definitely a bug, though.

**Comment by Guido07111975** [comment]
You mean it should NOT jump to the preview, because visually impaired users are not able to work with the pencils properly?

**Comment by KTS915** [comment]
Yes, exactly. The fact that the pencil icons use `title` attributes is a good clue, because they should not normally be used for accessibility reasons. But, in this case, it's not expected that users who have to use a keyboard will be accessing the pencil icons, so utilizing the `title` attribute to provide a visual pop-up for sighted users is actually a good idea. The fact that WP allowed tabbing all over the place is just another piece of evidence that shows they didn't pay much attention to accessibility issues when building the Customizer.

**Comment by Guido07111975** [comment]
Closed because no 2 is fixed in #2564.
